nexus-repository-helm-it/pom.xml

```diff
@@ -86,6 +86,17 @@
       <version>${nxrm-version}</version>
       <type>zip</type>
       <scope>test</scope>
+      <exclusions>
+        <!-- Avoid leaking older r plugins from distro -->
```

Comment:
should be helm instead of r

Reply:
https://github.com/sonatype-nexus-community/nexus-repository-helm/pull/134/commits/e38b5d4a6ff1e5b0935c04e39ce5ac3cb78784d9